docs/api.md

```diff
@@ -81,12 +81,14 @@ It requires at least one parameter to update.
 ```json
 {
   "title": "A new Subscriber list title",
+  "description": "A new subscriber list description",
 }
 ```
 
 The following fields are accepted:
 - title: The title of this particular list, which will be shown to the user;
   email sent to a user;
+- description: A description of the content this list represents, shown to the user in some emails. 
```

Comment:
I don't love this explanation. Suggestions for a better one are welcome!

Reply:
It's actually *not* used in any of the email bodies that email-alert-api generates (email-alert-service constructs the unpublishing email bodies).  Which does make this field a bit weird, as it's essentially arbitrary extra text data, passed over when the list is created, with no specific use...

Reply:
Yeah true. It will eventually end up in some of the emails (we'll pass it back to the service, the service will construct the email body, and then send the email through the API), but it's more of a case of the API database being the best place to store something we'll need later.

 I looked back at the description the [last time it existed ](https://github.com/alphagov/email-alert-api/pull/1527/files#diff-9eddf4dc51bf6a0125ca7fb094468ad284112270385c41cebce4f8f0a29620abL46) which explains the rogue `email sent to a user;` on line 90, but otherwise isn't very helpful.
 
 I think I'm going to go with: 
 "A description of the content this list represents, used by [email-alert-service](https://github.com/alphagov/email-alert-service) to construct emails when a page is unpublished."
 
 That'll at least give people somewhere to look for more information